lib/active_model/serializer/configuration.rb

```diff
@@ -7,6 +7,8 @@ module Configuration
       included do |base|
         base.config.array_serializer = ActiveModel::Serializer::ArraySerializer
         base.config.adapter = :flatten_json
+        base.config.enabled_adapters_by_media_type = true
```

Comment:
This should be false by default, as the defined serializers may not play well with all possible adapters.